spec/spec_helper.rb

```diff
@@ -10,7 +10,7 @@
 end
 
 SimpleCov.start("rails") do
-  add_filter "/lib/mi_bridges/"
+  # add_filter "/lib/mi_bridges/"
```

Comment:
What about something like this as a comment?

```
SimpleCov.start("rails") do
  # To exclude files from coverage analysis:
  # add_filter "/app/path/to/directory/"
end
```

Reply:
I commented this rather than deleted because I wasn't sure if we should exclude or not (see PR description). What do you think we should do?

Reply:
Either way is fine, I'm a 1/10 on my suggestion. I trust your discretion.

Reply:
the question I was (not very clearly) asking was whether you think we should continue to filter out `lib` in our test coverage? downside is that our test coverage is still low when we include it, upside is that it encourages us to add more tests when possible.

Reply:
We should not continue to filter. Let's keep testing, I think!